cse/indexing/DeltaPostingListIndex.py

```diff
@@ -34,7 +34,8 @@ def insert(self, pointer, commentId, tf, positions):
         if pointer not in self.__postingLists:
             self.__postingLists[pointer] = []
         self.__postingLists[pointer].append((commentId, tf, positions))
-        self.__postingLists[pointer].sort()
+        # should already be sorted:
+        #self.__postingLists[pointer].sort()
```

Comment:
you assume that `insert` is called in a "sorted way". 

Why is it clear that `insert` will always be called in this sorted way?

Reply:
I'm completly with you to reduce sort operations. But currently I'm not convinced that ths will result in a allways sorted list

Reply:
In the delta index I don't care if the posting lists are sorted. They will get sorted in the merge-step anyway.

Reply:
ok